src/main/kotlin/day6/Basic.kt

```
@@ -0,0 +1,21 @@
+package org.example.day6
+
+class Wand(
+    name: String
+) {
+    var name = require(name.length > 3)
+    fun attack() {
+
+    }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)